.circleci/config.yml

```diff
@@ -69,10 +69,6 @@ jobs:
 
     - run: ./architect deploy
 
-    - run: |
-        sed -i 's/version:.*/version: 1.0.0-'"${CIRCLE_SHA1}"'/' helm/cert-exporter-chart/Chart.yaml
```

Comment:
I don't follow, I created the sec magic to have a deployment to stable in the `1.0.0-sha` form, using architect deploy like this will cause you to always push 1.0.0 to stable I believe.

Reply:
Architect will try to replace the version in `chart.yaml` with the sha, but that version doesn't have the sha placeholder here: https://github.com/giantswarm/cert-exporter/blob/master/helm/cert-exporter-chart/Chart.yaml#L2

Reply:
OK thanks that helps. But is the sed magic working for chart-operator? 

The stable channel for chart-operator has version `0.2.4` which I don't get.

Reply:
What you are missing in this project is having the sed magic twice, to be precise on the `deploy` part at the very bottom - see `chart-operator` https://github.com/giantswarm/chart-operator/blob/master/.circleci/config.yml#L101-L116

On the stable channel stuff: Could it be that we fucked up there and never released to all envs? There hasn't been a new version pushed to stable for 19 days, which matches up with the last time we were all green in the master build: https://circleci.com/gh/giantswarm/chart-operator/tree/master